IonDotnet/Builders/IonLoader.cs

```diff
@@ -29,8 +28,8 @@ private IonLoader(ReaderOptions options)
         /// Load a string of Ion text.
         /// </summary>
         /// <param name="ionText">Ion text string.</param>
-        /// <returns>An <see cref="IonDatagram"/> tree view.</returns>
-        internal IonDatagram Load(string ionText)
+        /// <returns>An <see cref="IIonValue"/> tree view.</returns>
+        public IIonValue Load(string ionText)

```

Comment:
I think it would make more sense use `IIonDatagram` instead `IIonValue`. That will make it more clear to the user which methods are valid on values returned by `Load`.

Reply:
Just to be clear, we're talking about this and all other overloads of ``Load()`` method?

Reply:
Yes.

Reply:
Done